.github/workflows/airgap-test.yml

```diff
@@ -0,0 +1,44 @@
+name: Airgap
+on:
+  - push
+  - pull_request
+
+# Declare default permissions as read only.
+permissions: read-all
+
+jobs:
+  airgap-e2e-test:
```

Comment:
I would prefer to implement this as part of the existing `make e2e-test` target or a new `make e2e-airgap`, so it can be run locally.
We can leave "install kwctl" as a GHA step.

Reply:
I think it's better to leave that here. Running the test is cheap and we end up doing that on each PR

Reply:
I would leave it as it is as there are a few things that work in a different way than the current `e2e-test`:
- We need to run a local registry. GHA is creating the registry with a docker image. 
- Scripts are using [kwctl installed](https://github.com/raulcabello/kwctl/blob/save-load-script/scripts/kubewarden-save-policies.sh#L44) in your machine unlike e2e test that uses cargo run. 

I don't expect many changes to airgap in the future once it is finished, and GHA could be run locally with https://github.com/nektos/act

Reply:
let's leave it as it is then :).